src/routes/analysis/table/+page.svelte

```diff
@@ -0,0 +1,365 @@
+                x: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+                y: [11, 13, 15, 17, 14, 18, 21, 12, 14, 16],
+                color: "#6C3082",
+            },
+        },
+    ])
+    let blueTeams: TeamAlliance[] = $state([
+        {
+            teamColor: "bittersweet",
+            allianceNumber: "blue1",
+            teamNumber: data.blue1,
+            coral: 11,
+            algae: 12,
+            ability: "ability1",
+            algaeData: {
+                name: data.blue1,
+                x: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+                y: [12, 14, 11, 13, 15, 17, 14, 16, 18, 21],
+                color: "#FF6663",
+            },
+            coralData: {
+                name: data.blue1,
+                x: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+                y: [17, 14, 16, 18, 21, 12, 14, 11, 13, 15],
+                color: "#FF6663",
+            },
+        },
+        {
+            teamColor: "crayola_orange",
+            allianceNumber: "blue2",
+            teamNumber: data.blue2,
+            coral: 21,
+            algae: 22,
+            ability: "ability2",
+            algaeData: {
+                name: data.blue2,
+                x: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+                y: [12, 14, 16, 18, 21, 11, 13, 15, 17, 14],
+                color: "#FF8547",
+            },
+            coralData: {
+                name: data.blue2,
+                x: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+                y: [15, 17, 14, 16, 18, 21, 12, 14, 11, 13],
+                color: "#FF8547",
+            },
+        },
+        {
+            teamColor: "imperial_red",
+            allianceNumber: "blue3",
+            teamNumber: data.blue3,
+            coral: 31,
+            algae: 32,
+            ability: "ability3",
+            algaeData: {
+                name: data.blue3,
+                x: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+                y: [12, 15, 17, 14, 14, 16, 18, 21, 11, 13],
+                color: "#FF4848",
+            },
+            coralData: {
+                name: data.blue3,
+                x: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+                y: [11, 13, 15, 17, 14, 18, 21, 12, 14, 16],
+                color: "#FF4848",
+            },
+        },
+    ])
+
+    import * as Dialog from "$lib/components/ui/dialog/index.js"
+</script>
+
+<div
+    class="m-auto mb-2 grid h-96 w-full grid-cols-4 gap-1 border-2 border-solid border-imperial_red p-1"
+>
+    <div class="col-span-1 row-span-4 grid grid-rows-4 gap-1">
+        <div
+            class="row-span-1 w-full content-center bg-imperial_red text-center"
+        >
+            Team
+        </div>
+        <Dialog.Root>
+            <Dialog.Trigger
+                class="row-span-1 w-full content-center bg-gunmetal text-center text-imperial_red"
+                >Coral</Dialog.Trigger
+            >
+            <Dialog.Content
+                class="h-96 w-96 border-2 border-solid border-imperial_red bg-gunmetal"
```

Comment:
```suggestion
                class="size-96 border-2 border-solid border-imperial_red bg-gunmetal"
```
but not hard coding the height/width would be best